zasady_konstrukcji_api.rst

```diff
@@ -0,0 +1,254 @@
+﻿Zasady konstrukcji API
+======================
+do wewnętrznego użycia w EZD oraz dla zewnętrznych aplikacji — rekomendacje 
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+
+
+Spis rzeczy
+-----------
+
+ **API dla Java**
+
+ **API dla REST**
+
+ **API dla JavaScript**
+
+
+API dla Java
+------------
+
+**Projektuj przewidując rozwój**
+
+ Jeśli API jest cokolwiek warte – będzie ewoluować.   Z drugiej strony, jeśli coś w API już jest, to zapewne będzie musiało zostać.

```

Comment:
Zwykłe wcięcia oznaczają [cytaty blokowe](http://docutils.sourceforge.net/docs/user/rst/quickref.html#block-quotes) w formacie reStructuredText, co nie odpowiada faktycznej semantyce takich fragmentów. Powinniśmy korzystać albo z zagnieżdżonych nagłówków i zwykłych akapitów (być może bardziej pasuje to w tym przypadku), albo z [list definicyjnych](http://docutils.sourceforge.net/docs/user/rst/quickref.html#definition-lists).